src/confdb/confdb.h

```diff
@@ -154,6 +154,8 @@
 #define CONFDB_DEFAULT_SSH_KNOWN_HOSTS_TIMEOUT 180
 #define CONFDB_SSH_CA_DB "ca_db"
 #define CONFDB_DEFAULT_SSH_CA_DB SYSCONFDIR"/pki/nssdb"
+#define CONFDB_SSH_USE_CERT_KEYS "ssh_use_certificate_keys"
+#define CONFDB_DEFAULT_SSH_USE_CERT_KEYS true
```

Comment:
In general I agree with the option being enabled by default, I just wonder what is your opinion on enabling this also in conservative downstreams? (We can take this question off github as well)

Reply:
Actually, this was probably not a very good question. I thought that the difference this option is bringing is the verification, but looking at the code that this new code supersedes, the old code did also call the OCSP responder.

So the behaviour should be mostly the same as far as the user is concerned..

Reply:
yes, currently the generation of the ssh keys from certificates is unconditionally on. This comes from the time where for plain Smartcard authentication the full certificate had to be available as well. So it made sense to have the same behavior for ssh.

But new with certificate mapping and matching rules this might be different and even if the certificate is available in the user entry some users might not want to use it for ssh.